airbyte-integrations/connectors/source-asana/source_asana/manifest.yaml

```diff
@@ -0,0 +1,2566 @@
+    $ref: "#/definitions/base_stream"
+    name: "tasks"
+    primary_key: "gid"
+    schema_loader:
+      type: InlineSchemaLoader
+      schema:
+        $ref: "#/definitions/tasks_schema"
+    $parameters:
+      path: "/projects/{{ stream_slice['project_id'] }}/tasks"
+      extractor_path: "data"
+    retriever:
+      $ref: "#/definitions/retriever"
+      partition_router:
+        $ref: "#/definitions/projects_partition_router"
+
+  tasks_partition_router:
+    type: SubstreamPartitionRouter
+    parent_stream_configs:
+      - stream: "#/definitions/tasks_stream"
+        parent_key: "gid"
+        partition_field: "task_gid"
+
+  stories_compact_stream:
+    $ref: "#/definitions/base_stream"
+    name: "stories_compact"
+    primary_key: "gid"
+    schema_loader:
+      type: InlineSchemaLoader
+      schema:
+        $ref: "#/definitions/stories_compact_schema"
+    $parameters:
+      path: "/tasks/{{ stream_slice['task_gid'] }}/stories"
+      extractor_path: "data"
+    retriever:
+      $ref: "#/definitions/retriever"
+      partition_router:
+        $ref: "#/definitions/tasks_partition_router"
+
+  stories_compact_partition_router:
+    type: SubstreamPartitionRouter
+    parent_stream_configs:
+      - stream: "#/definitions/stories_compact_stream"
+        parent_key: "gid"
+        partition_field: "story_gid"
+
+  stories_stream:
+    $ref: "#/definitions/base_stream"
+    name: "stories"
+    primary_key: "gid"
+    schema_loader:
+      type: InlineSchemaLoader
+      schema:
+        $ref: "#/definitions/stories_schema"
+    $parameters:
+      path: "/stories/{{ stream_slice['story_gid'] }}"
+      extractor_path: "data"
+    retriever:
+      $ref: "#/definitions/retriever"
+      paginator:
+        $ref: "#/definitions/paginator_base"
+      partition_router:
+        $ref: "#/definitions/stories_compact_partition_router"
+
+  workspaces_stream:
+    $ref: "#/definitions/base_stream"
+    name: "workspaces"
+    primary_key: "gid"
+    schema_loader:
+      type: InlineSchemaLoader
+      schema:
+        $ref: "#/definitions/workspaces_schema"
+    $parameters:
+      path: "/workspaces"
+      extractor_path: "data"
+
+  workspaces_partition_router:
+    type: SubstreamPartitionRouter
+    parent_stream_configs:
+      - stream: "#/definitions/workspaces_stream"
+        parent_key: "gid"
+        partition_field: "workspace_gid"
+
+  tags_stream:
+    $ref: "#/definitions/base_stream"
+    name: "tags"
+    primary_key: "gid"
+    schema_loader:
+      type: InlineSchemaLoader
+      schema:
+        $ref: "#/definitions/tags_schema"
+    $parameters:
+      path: "/workspaces/{{ stream_slice['workspace_gid'] }}/tags"
+      extractor_path: "data"
+    retriever:
+      $ref: "#/definitions/retriever"
+      partition_router:
+        $ref: "#/definitions/workspaces_partition_router"
+
+  users_stream:
+    $ref: "#/definitions/base_stream"
+    name: "users"
+    primary_key: "gid"
+    schema_loader:
```

Comment:
Getting the following extra fields when running `discover` command:

- `path`
- `extractor_path`
- `$parameters`